.pre-commit-config.yaml

```diff
@@ -14,6 +14,9 @@ repos:
             )$
     -   id: check-added-large-files
     -   id: check-json
+    -   id: detect-aws-credentials
+    -   id: detect-private-key
+        exclude: data_dir/ssl_conf/*
```

Comment:
@amoskong do we need all those keys here ? or now everything is getting generated ? 
or even, why not all the needed keys are generated ?